File: Document Intelligence/Connect.py
from azure.core.credentials import AzureKeyCredential

from azure.ai.documentintelligence import DocumentIntelligenceClient

from azure.ai.documentintelligence.models import AnalyzeResult

 

endpoint = "<your-endpoint>"

key = "<your-key>"

 

docUrl = "<url-of-document-to-analyze>"

 

document_analysis_client = DocumentIntelligenceClient(endpoint=endpoint,

    credential=AzureKeyCredential(key))

 

poller = document_analysis_client.begin_analyze_document(

        "prebuilt-layout", AnalyzeDocumentRequest(url_source=docUrl

    ))

result: AnalyzeResult = poller.result()

File: Document Intelligence/Connect.cs
using Azure;

using Azure.AI.DocumentIntelligence;

 

string endpoint = "<endpoint>";

string key = "<access-key>";

AzureKeyCredential cred = new AzureKeyCredential(key);

DocumentIntelligenceClient client = new DocumentIntelligenceClient (new Uri(endpoint), cred);

 

Uri fileUri = new Uri ("<url-of-document-to-analyze>");

 

var content = new AnalyzeDocumentContent()

{

    UrlSource = fileUri

};
